docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -68,21 +68,24 @@ If you are using {agent} with link:{serverless-docs}[{serverless-full}], note th
 
 [discrete]
 [[elastic-agent-installation-minimum-requirements]]
-== Minimum requirements  
+== Resource requirements
+
+The {agent} resources consumption is influenced by the number of integration and the environment its been running on.
+
+Using our lab environment as an example, we can observe the following resource consumption:
 
-The following are the minimum system requirements for installing {agent}.
 // lint ignore mem
 [discrete]
 === CPU and RSS memory size
 
-// lint ignore 2vcpu 1gb
-Minimum requirements have been determined by running the {agent} on a GCP `e2-micro` instance (2vCPU/1GB).
-The {agent} used the default policy, running the system integration and self-monitoring.
-
+// lint ignore 4 vCPU 16.0 GiB We tested in AWS m7i.xlarge instance type that has 4 vCPUs, 16.0 GiB of memory and up to 12.5 Gibps of bandwidth. Ingesting a single log file, running with the throughput preset with self-monitoring enabled.
```

Comment:
```suggestion
// lint ignore 4 vCPU 16.0 GiB 
We tested using an AWS `m7i.xlarge` instance type with 4 vCPUs, 16.0 GB of memory, and up to 12.5 Gbps of bandwidth.
The tests Ingested a single log file, running with the throughput preset with self-monitoring enabled.
```

@alexsapran, Thanks for adding this! I hope you don't mind a comment while the PR is still a draft. Here's how the output will render:

![Screenshot 2024-10-31 at 10 05 09 AM](https://github.com/user-attachments/assets/7aaa63f9-e463-4959-b200-cc1e32b5ef4a)

Reply:
No, not at all. I was hoping to render like that and keep the previous table format. 

If we feel we should adjust to something else, let me know, I'm happy to adjust.

Reply:
Looks good! My only suggestion is to rephrase line 81 a bit:

```
We tested using an AWS `m7i.xlarge` instance type with 4 vCPUs, 16.0 GB of memory, and up to 12.5 Gbps of bandwidth.
The tests ingested a single log file, running with the throughput preset with self monitoring enabled.
```

Apart from that I think it's perfect! 👍